docs/dev/new_check_howto.md

```diff
@@ -166,7 +169,7 @@ def test_config():
     c.check({'url': 'http://foobar', 'search_string': 'foo'})
 ```
 
-The cookiecutter template has already setup `tox` to run tests located at `my_check/tests`. Run the test:
+The scaffolding has already set up `tox` to run tests located at `my_check/tests`. Run the test:
 
 ```
 cd my_check && tox
```

Comment:
Can we please update this too? Simply:

`ddev -e test my_check`

no matter what directory one is in.

Reply:
Ooh, that's a nice feature!